schema/rpc.fbs

```diff
@@ -521,3 +521,22 @@ table SetPauseTrackingRequest {
 
 /// Clears mounting reset data, defaulting to the manually set mounting orientations
 table ClearMountingResetRequest {}
+
+/// Common folders often used in computers for storing files
+enum ComputerDirectory: uint8 {
+    DOCUMENTS,
```

Comment:
```suggestion
    Documents,
```

This ain't java

Reply:
The enums on the rpc have pascal case and some others screaming snake case, what's the standard?

Reply:
Use PascalCase, if we let in SCREAMING_SNAKE that was an accident